src/app/pages/forms-and-validation/phone-number-format/phone-number-format.component.ts

```diff
@@ -0,0 +1,118 @@
+import { Component, OnInit } from '@angular/core';
+import { BreakpointObserver, Breakpoints, BreakpointState } from '@angular/cdk/layout';
+import { FormGroup, Validators, FormBuilder, AbstractControl } from '@angular/forms';
+import { StateService } from '../../../services/state.service';
+import { parsePhoneNumberFromString, AsYouType } from 'libphonenumber-js';
+
+@Component({
+    selector: 'app-phone-number-format',
+    templateUrl: './phone-number-format.component.html',
+    styleUrls: ['./phone-number-format.component.scss'],
+})
+export class PhoneNumberFormatComponent implements OnInit {
+    isSmall: boolean;
+    selectedPhoneNumber: any;
```

Comment:
Too much use of the `any` type throughout. Using any like this kind of defeats the point of using Typescript in the first place.

Reply:
Variable naming confusion: What is 'selectedPhoneNumber'? I don't see anywhere in the form where I am selecting a phone number.